lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -29,7 +29,7 @@ def source_dir
 
 def fixtures(category)
   begin
-    fixtures = YAML.load_file(".fixtures.yml")["fixtures"]
+    fixtures = YAML.load_file( ENV['FIXTURES_YML'] || '.fixtures.yml' )['fixtures']
```

Comment:
At the same time I'd like to get rid of the need to "get overly creative with it." If you have any public examples you could point me to, I'd be grateful.